### PR TITLE
Replace `whenHydrated` with `whenReady`

### PR DESCRIPTION
**Pull request by jakobe**
Clean up some test utility methods

### CODE REVIEW

File: libs/designsystem/src/lib/testing/test-helper.ts [hunk 1]
```diff
@@ -26,10 +30,15 @@ export class TestHelper {
     });
   }
 
+  /** Checks for the Ionic Web Component being hydrated, ie. the Shadow DOM is ready for query */
```
Comment (askarby) [inline libs/designsystem/src/lib/testing/test-helper.ts:33]: Looks like a copy of the documentation for the `whenHasCssClass`-function, I'm guessing this function doesn't have the same behaviour?

Comment (jakobe) [reply libs/designsystem/src/lib/testing/test-helper.ts]: Actually it was the other way around ;)

**Review by askarby — DISMISSED** [review]
giphub

**Review by lydemann — DISMISSED** [review]
LGTM

**Review by hafstad — APPROVED** [review]

**Pull request merged**